httpdocs/js/webots-cloud.js

```diff
@@ -516,7 +516,9 @@ document.addEventListener('DOMContentLoaded', function() {
             if (data.error)
               modal.error(data.error);
             else if (!cancelled) {
-              fetch('/ajax/animation/create.php', {method: 'post', body: JSON.stringify({uploading: 0, uploadId: data.id})})
```

Comment:
The changes in this file don't seem to be related with the goal of this PR. Is that on purpose?
I am not convinced they are very useful.